core/src/main/res/values/strings.xml

```diff
@@ -618,119 +618,119 @@
         <item quantity="one">1 hour</item>
         <item quantity="other">%d hours</item>
     </plurals>
-    <string name="auto_enable_label">Auto-enable</string>
-    <string name="sleep_timer_enabled_label">Sleep timer enabled</string>
+    <string name="auto_enable_label">Auto</string>
+    <string name="sleep_timer_enabled_label">Sleep timer on</string>
 
     <!-- gpodder.net -->
-    <string name="gpodnet_taglist_header">CATEGORIES</string>
-    <string name="gpodnet_toplist_header">TOP PODCASTS</string>
-    <string name="gpodnet_suggestions_header">SUGGESTIONS</string>
-    <string name="gpodnet_search_hint">Search gpodder.net</string>
-    <string name="gpodnetauth_login_butLabel">Login</string>
-    <string name="gpodnetauth_encryption_warning">Password and data are not encrypted!</string>
+    <string name="gpodnet_taglist_header">Categories</string>
```

Comment:
It should just be fuzzy. If you about wasted time on part of translators, don't use TX.

Reply:
Weblate handles fuzzy strings. In practical terms no translator ever complained, nor is likely to given the bulk change made. In fact people are very happy about changes, because it shows someone cares.

"Special thanks" in "About" is all big versals for example.
It is better if everyone is on the same page as far as consistency goes, and that nobody gets ideas for where to insert this practice. It is hard enough to keep track of 800 strings without the extra entropy.